src/UploadedFileInterface.hh

```diff
@@ -30,6 +30,17 @@
 
 namespace Facebook\Experimental\Http\Message;
 
+enum UploadedFileState: int {
+	ERROR_NONE = 0;
```

Comment:
Maybe:

a) rename to UploadedFileResult and rename ERROR_NONE to SUCCESS
b) rename to UploadedFileError, remove ERROR_NONE, and make getError() return `?UploadedFileError`

Reply:
I'm ok with option b